Integrations/MimecastV2/MimecastV2.py

```diff
@@ -509,10 +531,109 @@ def create_policy_request(policy, option):
             'option': option
         }]
     }
+    response = http_request('POST', api_endpoint, str(payload))
+    if response.get('fail'):
+        return_error(json.dumps(response.get('fail')[0].get('errors')))
+    return response.get('data')[0]
+
+
+def update_policy(args):
+    """
+          Update the policy acoording to policy ID
+     """
+    headers = ['Policy ID', 'Description', 'Sender', 'Reciever', 'Bidirectional', 'Start', 'End']
+    contents = {}  # type: Dict[Any, Any]
+    context = {}
+    policies_context = {}  # type: Dict[Any, Any]
+    policy_obj, option, policy_id = get_arguments_for_policy_command("update", args)
+    empty_args_list = []
+    for arg, value in policy_obj.items():
+        if value == '':
+            empty_args_list.append(arg)
+    if option == '':
+        empty_args_list.append("option")
+    if len(empty_args_list) > 0:
+        policy_details = get_policy_request(policy_id)
+        for arg in empty_args_list:
+            if arg == "option":
+                option = policy_details[0]["option"].encode("utf-8")
+            else:
+                policy_obj[arg] = policy_details[0]["policy"][arg].encode("utf-8")
+    respone = update_policy_request(policy_id, policy_obj, option)
+    policy = respone.get('policy')
+    title = 'Mimecast Update Policy: \n Policy Was Updated Successfully!'
+    sender = policy.get('from')
+    reciever = policy.get('to')
+    description = policy.get('description')
+    contents = {
+        'Policy ID': policy_id,
+        'Description': description,
+        'Sender': {
+            'Group': sender.get('groupId'),
+            'Email Address': sender.get('emailAddress'),
+            'Domain': sender.get('emailDomain'),
+            'Type': sender.get('type')
+        },
+        'Reciever': {
+            'Group': reciever.get('groupId'),
+            'Email Address': reciever.get('emailAddress'),
+            'Domain': reciever.get('emailDomain'),
+            'Type': reciever.get('type')
+        },
+        'Bidirectional': policy.get('bidirectional'),
+        'Start': policy.get('fromDate'),
+        'End': policy.get('toDate')
+    }
+    policies_context = {
+        'ID': policy_id,
+        'Description': description,
+        'Sender': {
+            'Group': sender.get('groupId'),
+            'Address': sender.get('emailAddress'),
+            'Domain': sender.get('domain'),
+            'Type': sender.get('type')
+        },
+        'Reciever': {
+            'Group': reciever.get('groupId'),
+            'Address': reciever.get('emailAddress'),
+            'Domain': reciever.get('domain'),
+            'Type': reciever.get('type')
+        },
+        'Bidirectional': policy.get('bidirectional'),
+        'FromDate': policy.get('fromDate'),
+        'ToDate': policy.get('toDate')
+    }
```

Comment:
it matches the context of create, right?

Reply:
yes, same like create